### ISSUE TITLE
[FEATURE] Add clutch function

### ISSUE THREAD

**Issue by qsych** [CCBlueX/LiquidBounce#2091]
### LiquidBounce Branch

Nextgen

### Describe your feature request.

Add clutch function.
Like this(0:42 - 0:46): https://youtu.be/TB7FCOlOpf8?si=De4mZJrskni3Q9bE&t=42

**Comment by SenkJu** [comment]
Could you elaborate?

**Comment by qsych** [comment]
> Could you elaborate?

https://www.youtube.com/watch?v=p5xmE-9Zde0

**Comment by MukjepScarlet** [comment]
> Could you elaborate?

Auto place blocks to prevent falling into the void.

**Comment by VillainsRule** [comment]
lowkey priority 😔

**Comment by TssCode** [comment]
bump
auto-clutch like Slinky has, will be very very handy

**Comment by MukjepScarlet** [comment]
tbh you can do it by quickly enable scaffold (normal), I've tried it many times

**Comment by SuicidalEeper** [comment]
Yeah it's basically just scaffold that enables only when you're falling into void or from some high up height
